### PR TITLE
new Templo pages added

### PR DESCRIPTION
**Pull request by xala3pa**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: espanol/templo/templo.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 Cuando estaban todos reunidos en las afueras del templo, el Consejero empezó a predicar como todas
 las tardes.
+
+[Alguien alzó la mano](manoArriba.md)
```
Comment (casey-collab) [inline espanol/templo/templo.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: espanol/templo/templo.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+
+[Se oyó un grito que decía ](grito.md)
```
Comment (casey-collab) [inline espanol/templo/templo.md]: This link is also invalid.

File: espanol/templo/manos/manoArriba.md [hunk 3]
```diff
@@ -0,0 +1 @@
+"Levantando su mano gritó herejes"
```
Comment (casey-collab) [inline espanol/templo/manos/manoArriba.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: espanol/templo/grito/grito.md [hunk 4]
```diff
@@ -0,0 +1 @@
+"Fuego en el Templo..."
```
Comment (casey-collab) [inline espanol/templo/grito/grito.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**